### PR TITLE
Fix #16: `condensedHeaderHeight` should update automatically

### LINKED ISSUE #16: mobile with widescreen not displaying correctly
Hi, I have a site at - https://crackling-heat-6713.firebaseapp.com/index.html#!/xmenu/home - that uses this, but when I view it on my htc m8 it looks good when viewing in portrait, but when I turn it to landscape, the background image and my title bleeds over into the content.

Thanks
Tim

### PR DESCRIPTION
**Pull request by blasten**

### CODE REVIEW

**Comment by blasten** [comment]
PTAL @frankiefu

File: paper-scroll-header-panel.html [hunk 1]
```diff
@@ -87,11 +87,13 @@
       right: 0;
       bottom: 0;
       left: 0;
-      box-sizing: border-box;
       -moz-box-sizing: border-box;
       -webkit-overflow-scrolling: touch;
+      box-sizing: border-box;
```
Comment (frankiefu) [inline paper-scroll-header-panel.html:92]: move `box-sizing: border-box` right below `-moz-box-sizing`

File: paper-scroll-header-panel.html [hunk 2]
```diff
@@ -286,6 +302,41 @@
     },
 
     /**
+     * The header's initial state
+     *
+     * @property HEADER_STATE_EXPANDED
+     * @type Number
+     */
+
```
Comment (frankiefu) [inline paper-scroll-header-panel.html]: remove newline

File: paper-scroll-header-panel.html [hunk 3]
```diff
@@ -286,6 +302,41 @@
...
+    HEADER_STATE_EXPANDED: 0,
+
+    /**
+     * The header's state when it's hidden.
+     *
+     * @property HEADER_STATE_HIDDEN
+     * @type Number
+     */
+
```
Comment (frankiefu) [inline paper-scroll-header-panel.html]: remove newline

File: paper-scroll-header-panel.html [hunk 4]
```diff
@@ -286,6 +302,41 @@
...
+    HEADER_STATE_CONDESED: 2,
+
+    /**
+     * The header's state when its progress is somewhere between
+     * the `hidden` and `condensed` state.
+     *
+     * @property HEADER_STATE_INTERPOLATED
+     * @type Number
+     */
+     HEADER_STATE_INTERPOLATED: 3,
```
Comment (frankiefu) [inline paper-scroll-header-panel.html]: fix indent

File: paper-scroll-header-panel.html [hunk 5]
```diff
@@ -315,6 +366,14 @@
       return this.$.mainContainer;
     },
 
+    get _headerMaxDelta() {
+      return (this.keepCondensedHeader) ? this._headerMargin : this.headerHeight;
```
Comment (frankiefu) [inline paper-scroll-header-panel.html]: should remove `(` and `)`

File: paper-scroll-header-panel.html [hunk 6]
```diff
@@ -352,27 +415,39 @@
...
       if (this.fixed) {
+        this._setHeaderState(this.HEADER_STATE_EXPANDED);
         this._transformHeader(null);
       } else {
-        this._scroll();
+
+        switch (this.headerState) {
+          case this.HEADER_STATE_HIDDEN:
+            this._transformHeader(this._headerMaxDelta);
+          break;
```
Comment (frankiefu) [inline paper-scroll-header-panel.html]: indent `break`

File: paper-scroll-header-panel.html [hunk 7]
```diff
@@ -352,27 +415,39 @@
...
+          case this.HEADER_STATE_CONDESED:
+            this._transformHeader(this._headerMargin);
+          break;
```
Comment (frankiefu) [inline paper-scroll-header-panel.html]: indent `break`

**Comment by frankiefu** [comment]
Can you also add `<body unresolved>` to all the demos?

**Comment by frankiefu** [comment]
Other than minor comments, LGTM :+1:

**Comment by blasten** [comment]
Thanks! I've addressed the feedback.

**Pull request merged**